src/pages/index.tsx

```diff
@@ -13,12 +13,28 @@ export function Pages() {
     {
       path: '/embalm',
       element: <EmbalmPage />,
-      label: 'Embalm',
+      noBg: true,
```

Comment:
even though chakra called it `bg`, I still might name this noBackground. So it might be less confusing looking at

`px={route.noBackground ? 0 : 5}`

but it's also fine like this

Reply:
Nah you're right. Updating.